dds_cli/utils.py

```diff
@@ -119,7 +119,8 @@ def create_table(
             *[
                 rich.markup.escape(
                     dds_cli.utils.format_api_response(
-                        str(row[x]) if ints_as_string and isinstance(row[x], int) else row[x], x
+                        row[x],
```

Comment:
We probably need to have this line as it was, it's used in a few other places and it affects the formatting of the ints and may even cause some issues with some other commands. 
```suggestion
                        str(row[x]) if ints_as_string and isinstance(row[x], int) else row[x], x
```

And then you could call the `create_table` function with `ints_as_string=False` instead. I tried this and it worked, I may be missing something though.

Reply:
Yes, I also thought `ints_as_string=False` is the next option. Not sure why I didn't go for it in the first place.